nanome_loaders/_WebLoaderServer.py

```diff
@@ -294,15 +298,15 @@ def do_DELETE(self):
             parsed_url = urlparse(self.path)
             file = parsed_url.path[1:]
             file = urllib.parse.unquote(file)
+            file = os.path.join(file_dir, file)
         except:
             Logs.warning("Error trying to parse request:\n", traceback.format_exc())
             self._send_json_error(200, "Parsing problem")
             return
 
         try:
-            if file != "":
-                if _WebLoaderServer.file_filter(file): # Make sure file to delete is a molecular file
-                    os.remove(file)
+            if file != "" and _WebLoaderServer.file_filter(file): # Make sure file to delete is a molecular file
```

Comment:
This condition might not work anymore, since you add the file_dir path.
Best solution is probably to add a if file != "" before line 301